virtual.py

```diff
@@ -28,5 +28,21 @@ def wishMe():
     else:
         speak("Hello,Good Evening")
         print("Hello,Good Evening")
+        
+        
+def takeCommand():
+    r=sr.Recognizer()
+    with sr.Microphone() as source:
+        print("Listening...")
+        audio=r.listen(source)
+
+        try:
+            statement=r.recognize_google(audio,language='en-in')
+            print(f"user said:{statement}\n")
+
+        except Exception as e:
+            speak("Pardon me, please say that again")
+            return "None"
+        return statement
```

Comment:
_Good work @santosh-sudo_
**Have good command on python and git**
@ayushii18 see if any mistakes had done ..

Reply:
No issues @shivamkumar0611 
we'll do some more coding from tommorow